### DISCUSSION TITLE
Action on right click?

### DISCUSSION THREAD

**Discussion by Xwilarg** [boardgameio/boardgame.io#1111] [Q&A] [answered]
Hey, is there any way to do a specific action when the user is doing a right click on a tile?

**Comment by jonaswre** [comment, marked as answer]
Right click depends on the environment you are running in. (Browser vs React native) 

This is just the first article I found when googling. 

https://www.pluralsight.com/guides/how-to-create-a-right-click-menu-using-react

Instead of drawing the context menu you could just execute a move.
